auth_admin_passkey/model/res_users.py

```diff
@@ -1,55 +1,40 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Admin Passkey module for OpenERP
-#    Copyright (C) 2013-2014 GRAP (http://www.grap.coop)
-#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright (C) 2013-2014 GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 import datetime
 
 from openerp import SUPERUSER_ID
-from openerp import pooler
+from openerp import registry
 from openerp import exceptions
-from openerp.osv.orm import Model
+from openerp import models, api
```

Comment:
IMO it is inconsistent to perform all the other `openerp` imports on one line, then do two here. Please either move the `api` import into its own line or, preferably, group them in one line.